code/modules/mob/living/simple_animal/abnormality/waw/snow_whites_apple.dm

```diff
@@ -77,6 +79,12 @@
 	..()
 
 /mob/living/simple_animal/hostile/abnormality/snow_whites_apple/death(gibbed)
```

Comment:
You might want to move some actions to Destroy() instead, since it *could* get deleted without dying.

Reply:
done